src/Resources/views/CRUD/action_buttons.html.twig

```diff
@@ -9,7 +9,7 @@ file that was distributed with this source code.
 
 #}
 {% spaceless %}
-    {% for item in admin.getActionButtons(action, object|default(null)) %}
+    {% for item in admin.getActionButtons(action, (object is defined)?object:null ) %}
```

Comment:
please add spaces around both `?` and `:`. Also, this might be simplified into `object|defaut(null)`, right?

Reply:
Ok for spaces but not for the `default` filter.

Twig doc about `empty` (which is called from `default`):
> # empty
> *New in version 2.3*: Support for the `__toString()` magic method has been added in Twig 2.3.
> `empty` checks if a variable is an empty string, an empty array, an empty hash, exactly `false`, or exactly `null`.
> For objects that implement the `Countable` interface, empty will check the return value of the `count()` method.
> For objects that implement the `__toString()` magic method (and not `Countable`), it will check if an empty string is returned.

Interresting part:
> For objects that implement the `__toString()` magic method (and not `Countable`), it will check if an empty string is returned.

So `object is defined ? object : null` is the right way to do it.